web/themes/custom/caesar/templates/patterns/atoms/button/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@caesar/a-button",
+  "version": "1.0.0",
+  "license": "MIT",
+  "dependencies": {
+    "@splidejs/splide": "^4.1.4"
```

Comment:
Please remove it, there's no wait to track dependencies for components which use it

Also no way to maintain (bump) components which using it

And there's no way to document how inter-dependencies should be upgraded according to semver.org

Reply:
this is new branch based on current one with bumping of version
https://github.com/skilld-labs/ui-patterns-kaizen/pull/7/files

more info:
https://github.com/lerna/lerna/tree/main/commands/version#semver-bump

Reply:
Dependencies not changed yet. Gonna research it later.